dags/openshift_nightlies/config/benchmarks/hosted-control-plane-p75.json

```diff
@@ -1,13 +1,13 @@
 {
     "benchmarks": [
         {
-            "name": "cluster-density-ms-p75",
+            "name": "cluster-density-v2-p75",
             "workload": "kube-burner-ocp-wrapper",
             "trigger_rule": "all_done",
             "command": "./run.sh",
             "env": {
-                "WORKLOAD": "cluster-density-ms",
-                "ITERATIONS": "75",
+                "WORKLOAD": "cluster-density-v2",
+                "ITERATIONS": "63",
```

Comment:
is 63 iterations the new 75th percentile number roughly? if not we should likely change the name to p63 or something to match